From 8a270d6dd40ac43c98c948fe9d10e5bcb4aa3d41 Mon Sep 17 00:00:00 2001
From: Jean-Baptiste Kempf <jb@videolan.org>
Date: Tue, 1 Sep 2015 16:00:59 +0200
Subject: [PATCH] Fix crashes with some DVDs on describe_title

This is a stop-over, but the actual issue is not fixed
---
 src/searching.c |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/searching.c b/src/searching.c
index 70c0f85..f638b61 100644
--- a/src/searching.c
+++ b/src/searching.c
@@ -644,6 +644,11 @@ uint32_t dvdnav_describe_title_chapters(dvdnav_t *this, int32_t title, uint64_t
   if(!tmp)
     goto fail;
 
+  if(!ptt) {
+      printerr("ptt NULL");
+      goto fail;
+  }
+
   length = 0;
   for(i=0; i<parts; i++) {
     uint32_t cellnr, endcellnr;
-- 
1.7.10.4

